webpack.config.js

```diff
@@ -66,6 +67,10 @@ const config = {
       template: 'res/index.html',
       favicon: 'res/favicon.png',
     }),
+    new CopyWebpackPlugin([
+      { from: 'res/_headers' },
+      { from: 'res/_redirects' },
```

Comment:
Where does this get copied to, during which command?

Reply:
That specific file gets put into `dist` during any webpack command. I could make it prod only. The syntax for the CopyWebpackPlugin is a little weird.

Reply:
How does Netlify know that it needs to serve the `dist` directory? Is that just a setting in its settings UI?

Reply:
Wondering if we should use this plugin instead of the manual "cp" in the various commands in package.json. (maybe in a cleaning future PR).

Reply:
@mstange see the "publish directory" in the options in netlify GUI:
![image](https://user-images.githubusercontent.com/454175/38497269-d83e9528-3c00-11e8-8c5a-fd46cff01ff4.png)

But I wonder if we should introduce a [netlify.toml file](https://www.netlify.com/docs/continuous-deployment/#deploy-contexts) instead, and specify this through the `publish` property. So that everything is contained in our repository.

Reply:
I'm finding the `toml` approach more confusing to implement compared to the UI. I'm going to keep it as is for now, we can always follow-up.